redisenterprise/datadog_checks/redisenterprise/check.py

```diff
@@ -0,0 +1,314 @@
+import sys
+from datetime import datetime, timedelta
+
+from requests.auth import HTTPBasicAuth
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+# from typing import Any
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'redisenterprise'
+
+
+class RedisenterpriseCheck(AgentCheck):
+    """RedisenterpriseCheck attempts to connect to the cluster and ensure the node is the master node"""
+
```

Comment:
```suggestion
    HTTP_CONFIG_REMAPPER = {'tls_verify': {'name': 'tls_verify', 'default': False}}
```
@maguec This remapper will set the default value you want for `tls_verify`, so you can remove `verifyssl` option

Reply:
@coignetp Paul - that fixes it!  Thanks for the great pointer